spec/spec_helper.rb

```diff
@@ -1,3 +1,6 @@
+java_import java.util.logging.Logger
+java_import java.util.logging.Level
```

Comment:
...if we add it back to client_spec

But I thought things like this belonged in the spec_helper